use_cases/eluc/prescriptors/nsga2/candidate.py

```diff
@@ -87,10 +85,13 @@ def record_state(self) -> dict:
         """
         if not isinstance(self.metrics, tuple):
             raise ValueError("Candidate has not been evaluated yet")
-        return {"gen": self.gen,
-                "id": self.cand_id,
-                "parents": self.parents,
-                "NSGA-II_rank": self.rank, # Named this to match ESP
-                "distance": self.distance,
-                "ELUC": self.metrics[0],
-                "change": self.metrics[1]}
+        cand_state = {"id": self.cand_id,
+                      "parents": self.parents,
+                      "NSGA-II_rank": self.rank, # Named this to match ESP
+                      "distance": self.distance,
+        }
```

Comment:
Now we save our candidate id as "id" in our logs. Not a huge fan of this change since "id" is a python builtin so we can't name our variables that but it matches ESP. This is still nicer than what I had before which was using gen and id as a double index.

Reply:
I don't think we have to match ESP anymore.
distance is the `NSGA-II_distance` though, right? Maybe we 
`id` could become `cid` for candidate id if needed. But dictionary keys don't conflict with python buildins so as long as we don't call our variables `id` we're fine.